SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -224,19 +225,15 @@
     <!-- **************************************************************************** -->
     <!-- Create the project-specific directories for conf and out -->
 
-    <!-- Calculate a project-specific directory names for conf and out -->
+    <!-- Create folder with unique name in the .sonarqube/out/ folder -->
+    <MakeUniqueDir Path="$(SonarQubeOutputPath)" Prefix="$(MSBuildProjectName)_">
```

Comment:
I wanted to suggest you to rename this class to use a long name but I can see that other tasks used here uses the `Dir` suffix so... all good.